deltacat/storage/model/locator.py

```diff
@@ -108,12 +109,25 @@ def hexdigest(self) -> str:
         for use in equality (i.e. two locators are equal if they have the same
         hexdigest) and inclusion in URLs.
         """
-        return sha1_hexdigest(self.canonical_string().encode("utf-8"))
+        # TODO uncomment for PR
+        # return sha1_hexdigest(self.canonical_string().encode("utf-8"))
+        return self._normalize_for_filesystem(self.canonical_string())
```

Comment:
This might be handy to enable/disable in general via a "developer mode" or "human readable" flag.